ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/AuthenticationPlugin.java

```diff
@@ -47,15 +47,15 @@ CompletableFuture<Principal> handleAuth(RoutingContext routingContext,
       WorkerExecutor workerExecutor);
 
   /**
-   * Retrieve the authorization token from the request. This is different from {@code handleAuth}
-   * since we need to expose the authorization token in order to provide forwarded inter-node
+   * Retrieve the authorization header from the request. This is different from {@code handleAuth}
+   * since we need to expose the authorization header in order to provide forwarded inter-node
    * requests the correct credentials.
    *
    * @param routingContext The routing context
-   * @return A String that is the authorization token that we can then use for forwarding
+   * @return A String that is the authorization header that we can then use for forwarding
    *        inter-node requests.
    */
-  default String getAuthToken(final RoutingContext routingContext) {
+  default String getAuthHeader(final RoutingContext routingContext) {
```

Comment:
Is there any chance that are other implementations of this interface?

E.g., is this a breaking change?

Reply:
technically it is, but I believe @nateab just rolled this out a week ago and the only implementation in our security plugins was reverted so there's no released code version with this change in it

Reply:
Nah I doubt it because I was the one who added this recently in https://github.com/confluentinc/ksql/pull/9239/files

Reply:
ah almog beat me to it!

Reply:
I take pride in my ✅ and felt the need to defend it! 😆